Makefile

```diff
@@ -383,15 +383,11 @@ ifeq ($(AUTODESPAWN), true)
 	@scripts/setup_integration_env.sh pulsar stop
 endif
 
-.PHONY: test-integration-splunk
-test-integration-splunk: ## Runs Splunk integration tests
+.PHONY: test-integration-splunk-7
```

Comment:
Also curious if the `7` target can work the same as the others where they use the wildcard style target.